fastlane/lib/fastlane/actions/app_store_build_info.rb

```diff
@@ -0,0 +1,198 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      LATEST_BUILD_NUMBER = :LATEST_BUILD_NUMBER
+      LATEST_VERSION_NUMBER = :LATEST_VERSION_NUMBER
+    end
+
+    class AppStoreBuildInfoAction < Action
+      def self.run(params)
+        require 'spaceship'
+
+        build_nr = get_build_number(params)
+
+        # Convert build_nr to int (for legacy use) if no "." in string
+        if build_nr.kind_of?(String) && !build_nr.include?(".")
+          build_nr = build_nr.to_i
+        end
+
+        Actions.lane_context[SharedValues::LATEST_BUILD_NUMBER] = build_nr
+      end
+
+      def self.get_build_number(params)
```

Comment:
This now gets build into so it should probably get renamed to `get_build_info`?

Reply:
renamed